### PR TITLE
Creating a data model 🌵 

### PR DESCRIPTION
**Pull request by evansnyamekye**
### Pull Request Description

Overview ⭕
This PR introduces the initial setup and database schema for the Blog Rails application. It addresses the project requirements as outlined below:

**Project Requirements**

- _Project Name:_ [Blog-App] 📴.
- _Git Initialization:_ This project has been initialized with `Git` ⚓.
- _Database:_ The application is configured to use a PostgreSQL `database` 🅱️.
- _Database Schema:_ The `database schema` has been designed based on the 
   provided ERD (Entity-Relationship Diagram) and project specifications 🪀.
- _Migration Files:_ Migration files have been created and executed to establish the `database` schema.

**Details**

- **Table and Column Names:** The table and column names in the database schema 
    match the ERD diagram provided in the project documentation.
- **Users Table Photo:** The `photo` column in the `users` table stores a link to an image.
- **Foreign Keys:** Foreign keys have been included in the appropriate tables to 
   establish 
-    data relationships.
- **Author Column in Posts Table:** The column holding the foreign key to the `users` 
   table in the `posts` table has been named `author_id`.
- **Indexes**: All columns that serve as foreign keys have corresponding indexes to 
   optimize database performance.

**Checklist**

- [ ]  Database schema design and migration files are accurate.
- [ ]  Foreign keys and indexes have been set up correctly.
- [ ]  The project adheres to coding and naming conventions.
- [ ]  All relevant ERD relationships are implemented.
- [ ]  Tests pass and have been added/updated accordingly.

Thank you 👏 for your review!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,135 @@
...
-* Database initialization
 
-* How to run the test suite
+- **Ruby**
 
-* Services (job queues, cache servers, search engines, etc.)
+### Key Features <a name="key-features"></a>
 
-* Deployment instructions
+- **Decode a Morse Code Message**
```
Comment (prolajumokeoni) [inline README.md]: - [ ] Kindly update the key features content to be related to this Blog app project 👍🏿

File: README.md [hunk 2]
```diff
@@ -1,24 +1,135 @@
...
+👤 **Evans Kofi Nyamekye**
+- GitHub: [evansnyamekye](https://github.com/evansnyamekye)
+- Twitter: [@nyamekye2131](https://twitter.com/nyamekye2131)
+- LinkedIn: [Evans Kofi Nyamekye](https://www.linkedin.com/in/evans-kofi-nyamekye-1980a4117/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Decode Longer Messages**
```
Comment (prolajumokeoni) [inline README.md]: - [ ] Kindly update the future features content to be related to this Blog app project 👍🏿

File: README.md [hunk 3]
```diff
@@ -1,24 +1,135 @@
...
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I want to thank Sadaf and Ben for always availing themselves of detailed project discussions. I thank Microverse for the chance to become a software developer.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (prolajumokeoni) [inline README.md:145]: - [ ] Kindly fix the link to your MIT license page when I click the link in this section it shows a 404 error page

File: README.md [hunk 4]
```diff
@@ -1,24 +1,135 @@
...
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/evansnyamekye/blog-app/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project...
```
Comment (prolajumokeoni) [inline README.md]: - [ ] It would be nice to update this sentence, you can ask people to star your project to show their support 👍🏿

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @evansnyamekye 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️  Database adapter is using Postgresql
✔️  Created migration files
 



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by gemmen29 — APPROVED** [review]
Hi @evansnyamekye, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**